integration-testing/src/test/java/org/triplea/lobby/server/db/dao/api/key/LobbyApiKeyDaoTest.java

```diff
@@ -10,24 +10,24 @@
 import java.util.Optional;
 import org.junit.jupiter.api.Test;
 import org.triplea.lobby.server.db.dao.DaoTest;
-import org.triplea.lobby.server.db.data.ApiKeyUserData;
 import org.triplea.lobby.server.db.data.UserRole;
+import org.triplea.lobby.server.db.data.UserWithRoleRecord;
 
 @DataSet(cleanBefore = true, value = "lobby_api_key/initial.yml")
 class LobbyApiKeyDaoTest extends DaoTest {
 
   private static final int USER_ID = 50;
 
-  private static final ApiKeyUserData EXPECTED_MODERATOR_DATA =
-      ApiKeyUserData.builder()
+  private static final UserWithRoleRecord EXPECTED_MODERATOR_DATA =
+      UserWithRoleRecord.builder()
           .userId(USER_ID)
           .username("registered-user")
           .role(UserRole.MODERATOR)
           .build();
-  private static final ApiKeyUserData EXPECTED_ANONYMOUS_DATA =
-      ApiKeyUserData.builder().username("some-other-name").role(UserRole.ANONYMOUS).build();
-  private static final ApiKeyUserData EXPECTED_HOST_DATA =
-      ApiKeyUserData.builder().role(UserRole.HOST).build();
+  private static final UserWithRoleRecord EXPECTED_ANONYMOUS_DATA =
+      UserWithRoleRecord.builder().username("some-other-name").role(UserRole.ANONYMOUS).build();
+  private static final UserWithRoleRecord EXPECTED_HOST_DATA =
+      UserWithRoleRecord.builder().role(UserRole.HOST).build();
```

Comment:
Same here with the suffix

Reply:
I'm hesitant. `EXPECTED_ANONYMOUS_RECORD` could be thought of as a record in the lobby_user table, that the data represents an `ANONYMOUS` user rather than a `UserWithRole`.  `EXPECTED_ANONYMOUS_USER_WITH_ROLE_RECORD` is too wordy (just a poor name there where we should be able to name this after what it represents and avoid baking in the type information).